app/services/rapid_ready_for_decision/views/hypertension_pdf.erb

```diff
@@ -0,0 +1,139 @@
+Hypertension Rapid Ready for Decision | Claim for Increase <br />
```

Comment:
Perhaps add a comment to remind developers that the html won't be interpreted/rendered strictly as html.